js/app.js

```diff
@@ -23,12 +19,12 @@ projects.forEach(function(projectObj) {
   $('#projects').append(projectObj.toHtml());
 });
 
-$('section').not('#home').hide();
-
-$('nav').on('click', '.tab', function(event){
-  var $activeDiv = $(this).attr('href');
-  event.preventDefault();
-  $activeDiv.show(800);
-  // $activeDiv.hide(1600);
-  console.log('I was clicked');
-});
+// $('section').not('#home').hide();
+//
+// $('nav').on('click', '.tab', function(event){
+//   var $activeDiv = $(this).attr('href');
+//   event.preventDefault();
+//   $activeDiv.show(800);
+//   // $activeDiv.hide(1600);
+//   console.log('I was clicked');
+// });
```

Comment:
Check your syntax for grabbing the $activeDiv - it may not be picking up what you think it should.